packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
```

Comment:
yeah that makes sense @greglo. `display: flex` is a little unintuitive because it primarily sets up positioning for its children, not itself -- other `display` values do the opposite.